src/System.Net.Http/tests/FunctionalTests/TestHelper.cs

```diff
@@ -155,6 +155,12 @@ public static void EnsureHttp2Feature(HttpClientHandler handler)
                 "_maxHttpVersion",
                 BindingFlags.NonPublic | BindingFlags.Instance);
             field_maxHttpVersion.SetValue(_settings, new Version(2, 0));
+
+            // Allow HTTP/2.0 via unencrypted socket if ALPN is not supported on platform.
+            FieldInfo field_allowPlainHttp2 = type_HttpConnectionSettings.GetField(
+                "_allowUnencryptedHttp2",
+                BindingFlags.NonPublic | BindingFlags.Instance);
+            field_allowPlainHttp2.SetValue(_settings, !PlatformDetection.SupportsAlpn);
```

Comment:
Doesn't this mean we're not running any tests in the default configuration?  I think that's a problem.

Reply:
We will for all OSX version and older Windows version.
Since the non-ssl version has very little platform dependencies I thought it would be sufficient.